internal/command/delta.go

```diff
@@ -70,26 +72,45 @@ func delta(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("preparing new deployment: %w", err)
 	}
 
-	// Create a new deployment delta
-	//
-	log.Print("Creating a new deployment delta...\n")
 	client, err := api.NewClient(apiUrl, apiToken, http.DefaultClient)
 	if err != nil {
 		return err
 	}
-	res, err := client.CreateDelta(context.Background(), orgID, appID, delta)
-	if err != nil {
-		return err
+
+	var res *ht.DeploymentDelta
+	if deltaUseId == "" {
+
+		log.Print("Creating a new deployment delta...\n")
+		res, err = client.CreateDelta(context.Background(), orgID, appID, delta)
+		if err != nil {
```

Comment:
This statement is duplicated. Should we move it outside the `if` block?

Reply:
sure thing 👍